packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -392,6 +394,12 @@ export class StatsAnalyzer extends EventsScope {
   filterAndParseGetStatsResults(statsItem: any, type: string, isSender: boolean) {
     const {types} = DEFAULT_GET_STATS_FILTER;
 
+    const reports = [];
+    statsItem.report.forEach((report) => reports.push(report));
+    this.successfulCandidatePair = reports.find(
```

Comment:
this seems a bit inefficient - we're traversing all the reports twice, first in line 398 then in line 399, we could just check the report.type and report.state in line 398, couldn't we?